Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1484,6 +1526,12 @@ <h3>
     b. Daten auf Serversystemen
 
 </h3>
+<p>
+    Im Falle der Aktualisierung eines digitalen COVID-Zertifikats (siehe Abschnitt 6.o) werden die
+    für die Aktualisierung an das Serversystem der App übermittelten digitalen COVID-Zertifikate
+    nach Abschluss des Aktualisierungsvorgangs wieder gelöscht. Der Hash-Wert der elektronischen
+    Signatur des aktualisierten digitalen COVID-Zertifikats wird nach X ZEITABSCHNITTE gelöscht.
```

Comment:
Hi  @LukasLechnerDev ,
in this line there is still the "365 days" missing.

Reply:
```suggestion
    Signatur des aktualisierten digitalen COVID-Zertifikats wird nach 365 Tagen gelöscht.
```